src/components/UserStats/Item.vue

```diff
@@ -1,12 +1,13 @@
 <template>
   <div class="flex flex-col items-center justify-center">
     <Label
-      v-if="preset === 'default'"
       preset="h3"
       class="text-like-green"
+      content-class="!items-end"
```

Comment:
```suggestion
      valign="bottom"
```

Reply:
I think v-align is not worked for this case, it can only work when adding `#prepend ` or  `#append `.

![image](https://user-images.githubusercontent.com/75730405/188300094-690b43e3-6fe6-4fc8-8339-87c1600e9190.png)
![image](https://user-images.githubusercontent.com/75730405/188300169-0871bd4c-ea0a-40e9-b0ab-81ab2a2b6c0b.png)

Reply:
Should be a mistake from Label, I think should change this [line](https://github.com/AuroraHuang22/liker-land/blob/afde72a633c32abfb98c9aae491f4d074d7b0ae8/src/components/Label.vue#L186) to `this.valignClass`